helm/hpcc/templates/thor.yaml

```diff
@@ -156,13 +164,7 @@ data:
           - name: {{ $agentAppType }}-%jobname
 {{- include "hpcc.addSecurityContext" (dict "root" $ "me" .) | indent 12 }}
 {{ include "hpcc.addImageAttrs" (dict "root" $ "me" .) | indent 12 }}
-            resources:
-              requests:
-                #request-memory
-                #request-cpu
-              limits:
-                #limit-memory
-                #limit-cpu
+{{- include "hpcc.addResources" (dict "me" $thorScope.agentResources) | indent 12 }}
```

Comment:
Didn't we settle on eclagentResources, managerResources and workerResources?
it may be fine, but 'agent' is a bit overloaded